### PR TITLE
WT-12074 Rename backup directory before removal.

### PR DESCRIPTION
**Pull request by sueloverso**

### CODE REVIEW

File: test/csuite/timestamp_abort/main.c [hunk 1]
```diff
@@ -489,7 +490,7 @@ backup_delete_old_backups(int retain)
     DIR *d;
     size_t len;
     int count, i, indexes[256], last_full, ndeleted;
-    char buf[256];
+    char buf[256], new[256];
```
Comment (pmacko86) [inline test/csuite/timestamp_abort/main.c:493]: I'd recommend that we use a different name, as `new` is a reserved word in C++, especially as there is an interest in making our C code C++-compatible in the future.

**Review by pmacko86 — COMMENTED** [review]
The change to rename the directory and then delete it looks good, except for the variable name.

Sue, you mentioned in the ticket that "one thing to keep in mind with this solution would be making sure if the kill happens during that time, that the renamed directories are also cleaned up." It would be nice to do, but it is also not necessary.

If you'd like to do this, this would be easy to do with `testutil_remove`, as it accepts globs just like the regular `rm -rf` command.

But if we decide that we don't have to do this final clean up, let us know, and I'll just LGTM as that part looks good.

File: test/csuite/timestamp_abort/main.c (hunk 1 above)
Comment (sueloverso) [reply test/csuite/timestamp_abort/main.c:493]: Fixed.

**Comment by sueloverso** [comment]
I answered your question in the ticket. The other issue is fixed (and will be committed soon)

**Review by pmacko86 — APPROVED** [review]
LGTM, after the variable name is fixed. Thank you!

**Pull request merged**